pkg/collector/collector.go

```diff
@@ -13,72 +13,93 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
- * Copyright 2018 Red Hat, Inc.
+ * Copyright 2019 Red Hat, Inc.
  */
 
 package collector
 
 import (
-	"encoding/xml"
-	"io/ioutil"
-	"os"
-	"strings"
+	"kubevirt.io/kubevirt-cpu-nfd-plugin/pkg/feature"
+	"kubevirt.io/kubevirt-cpu-nfd-plugin/pkg/file"
+	"kubevirt.io/kubevirt-cpu-nfd-plugin/pkg/util"
 )
 
-const usableNo string = "no"
+const (
+	usableNo string = "no"
+)
+
+var domCapabilitiesFilePath = "/etc/kubernetes/node-feature-discovery/source.d/virsh_domcapabilities.xml"
```

Comment:
can be `const`

Reply:
It is not constant, because in tests i need to change the path and const cannot be changed